json/authelia.json

```
@@ -0,0 +1,34 @@
+{
+    "name": "Authelia",
+    "slug": "authelia",
+    "categories": [
+        6
+    ],
+    "date_created": "2025-02-05",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": 9091,
+    "documentation": "https://www.authelia.com/integration/deployment/bare-metal/",
+    "website": "https://www.authelia.com/",
+    "logo": "https://camo.githubusercontent.com/bea2da4de8f6101f3d4f819585c23b01a08f5bcb20af70b900ac5d936879a99a/68747470733a2f2f7777772e61757468656c69612e636f6d2f696d616765732f61757468656c69612d7469746c652e706e67",
```

Authelia offers their branding icon directly on their website: 
https://www.authelia.com/reference/guides/branding/

```suggestion
    "logo": "https://www.authelia.com/images/branding/logo.png",
```